integration-cli/docker_utils.go

```diff
@@ -338,10 +339,11 @@ func sockRequest(method, endpoint string, data interface{}) (int, []byte, error)
 
 	res, body, err := sockRequestRaw(method, endpoint, jsonData, "application/json")
 	if err != nil {
-		b, _ := ioutil.ReadAll(body)
+		if body != nil {
```

Comment:
so this whole if block isn't needed at all since if `sockRequestRaw` errors out `body` is always nil so we can just `return -1, nil, err` :)